api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -7850,6 +7853,108 @@ message WorkloadIdentityDelete {
   ];
 }
 
+// WorkloadIdentityX509RevocationCreate is emitted when a
+// WorkloadIdentityX509Revocation is created.
+message WorkloadIdentityX509RevocationCreate {
+  // Metadata is a common event metadata
+  Metadata Metadata = 1 [
```

Comment:
Could we use "proper" proto style for the new messages?

* https://protobuf.dev/programming-guides/style/#field-names